test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
> hmmmm #2201 reminds me of older EmberJS code that heavily relied on state machines - which is what i think @jmeas is talking about. i agree that routes represent states.

You nailed it! Our router plans are heavily inspired by Ember's routing system.

> i'm assuming (requesting!!!!) that marionette's module system is also going away? please? :)

Hahaha. Yes, they are going away :) At least, they're being removed from the main codebase. The current code will be moved to a separate repository, so all of the people that follow Sulc's and @brian-mann's guides can easily migrate. @thejameskyle also made a fantastic micro-lib that re-creates the namespacing functionality of Modules in like 10 LoC. That guy's brilliant.